### PR TITLE
Add component to show selected timeseries in TimeseriesSearch

### PR DESCRIPTION
**Pull request by follan**
#123

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-130.gearboxjs.preview.cogniteapp.com

File: src/components/TimeseriesSearch/SelectedItems.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+export class SelectedItems extends React.Component<SelectedTimeseriesProps> {
+  static defaultProps = {
+    selectedItems: [],
+    onItemClose: null,
+  };
+
+  onClose = (item: Item) => {
+    const newItems = [...this.props.selectedItems].filter(
```
Comment (mrtraser) [inline src/components/TimeseriesSearch/SelectedItems.tsx]: For what you're creating new array for filtering it instead of using it directly from props?

Comment (follan) [reply src/components/TimeseriesSearch/SelectedItems.tsx]: No reason at all. Fixed now

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-130.gearboxjs.preview.cogniteapp.com

File: src/components/TimeseriesSearch/TimeseriesSearch.stories.tsx [hunk 2]
```diff
@@ -10,37 +10,45 @@ const timeseriesIds = timeseriesList.map(ts => ts.id);
...
+const setupMocks = () => {
+  sdk.Assets.list = async (
+    input: sdk.AssetListParams
+  ): Promise<sdk.AssetDataWithCursor> => {
+    action('sdk.Assets.list')(input);
+    return {
+      items: assetsList.map(
+        (a: sdk.Asset): sdk.Asset => {
+          return {
+            id: Number.parseInt(a.id.toString(), 10),
```
Comment (f1cognite) [inline src/components/TimeseriesSearch/TimeseriesSearch.stories.tsx]: Isn't `a.id` always a number?

Comment (follan) [reply src/components/TimeseriesSearch/TimeseriesSearch.stories.tsx]: Yes. Updated PR

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-130.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-130.gearboxjs.preview.cogniteapp.com

**Pull request merged**